config/environments/development.rb

```diff
@@ -61,5 +61,4 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
   config.debug_exception_response_format = :default
   # config.action_mailer.default_url_options = { host: ENV.fetch('HOST'), port: 3000 }
-  config.action_mailer.default_url_options = { host: ENV.fetch('HOST', 'localhost'), port: 3000 }
```

Comment:
hmm, why did you remove this line?

Reply:
I tried to configure "letter opener" to display messages, and then it turned out that I need to use SMS sending. It seems I have not returned everything to its original state.

Reply:
fixed it